docs/api/table/inputs.md

```diff
@@ -161,14 +165,14 @@ Default value: `[]`
 
 Row selection mode
 
-| Value                  | Description                                           | Default |
-| :--------------------- | :---------------------------------------------------- | ------- |
-| `undefined|false|null` | Rows cannot be selected                               | x       |
-| `"single"`             | One row can be selected at a time                     |         |
-| `"cell"`               | One cell can be selected at a time                    |         |
-| `"multi"`              | Multiple rows can be selected using Ctrl or Shift key |         |
-| `"multiClick"`         | Multiple rows can be selected by clicking             |         |
-| `"checkbox"`           | Multiple rows can be selected using checkboxes        |         |
+| Value          | Description                                           | Default |
+| :------------- | :---------------------------------------------------- | ------- | ----------------------- | --- |
+| `undefined     | false                                                 | null`   | Rows cannot be selected | x   |
```

Comment:
This change seems not to be related to the new feature. If not please remove.

Reply:
removed auto spacing change